pollux/lib/anoncreds/native-lib/helper_script_to_update.sh

```diff
@@ -1,24 +1,29 @@
 #!/usr/bin/env bash
 
+#  https://github.com/input-output-hk/anoncreds-rs/releases/download/0.4.0/library-darwin-aarch64-0ce097bbc9afa8a17a491248d75dcdf03f133dc4.tar.gz
 RELEASES=https://github.com/input-output-hk/anoncreds-rs/releases/download/
-VERSION=expose_presentation_methods_in_uniffi
-SHA="16a7178ca979f7643788f6ebf8189131ab922a53"
+VERSION="0.4.0"
+SHA="0ce097bbc9afa8a17a491248d75dcdf03f133dc4"
```

Comment:
Where do I get this value from in case I've to rebuild things?

Reply:
https://github.com/input-output-hk/anoncreds-rs/releases